architecture/trusted_applications.rst

```diff
@@ -170,27 +170,8 @@ property must be defined by the macro ``TA_STACK_SIZE`` in
 
 2. Property extensions
 ======================
-2.1 User Mode Flag
-==================
-``TA_FLAG_USER_MODE`` is a bit flag supported by ``TA_FLAGS``. This property
-flag is currently meaningless in OP-TEE. It may be set or not without impact on
-TA execution. All OP-TEE TAs are executed in user mode/level. Because of this we
-**do not** recommend to use this flag.
-
-2.2 DDR Flag
-============
-``TA_FLAG_EXEC_DDR`` is a bit flag supported by ``TA_FLAGS``. This property flag
-is currently meaningless in OP-TEE. Nevertheless it shall be set. It is a legacy
-property flag that aimed at targeting location for the TA execution, internal
-RAM or external DDR. Therefore all TAs must set ``TA_FLAG_EXEC_DDR`` in
-``TA_FLAGS`` in their ``user_ta_header_defines.h`` header file (see
-:ref:`user_ta_header_defines_h`).
 
-.. note::
-
-    This flag will soon be deprecated.
-
-2.3 Secure Data Path Flag
+2.1 Secure Data Path Flag
 =========================
```

Comment:
from https://github.com/OP-TEE/optee_docs/pull/5#issuecomment-471468593, looks like this should be changed into `-------`.

Reply:
and likely numbering `2.1` removed.

Reply:
Ah, I saw no warnings or anythings, so I just pushed this ... 2 seconds before you added this comment.

Reply:
Let's cross check and force push this last commit before merging something else.